src/app/jobs_workers/collect_main_files_worker.py

```
@@ -0,0 +1,158 @@
+    # Update job status to completed
+    jobs_service.update_job_status(job_id, "completed", result_file)
+
+    logger.info(
+        f"Job {job_id} completed: {result['summary']['updated']} updated, "
+        f"{result['summary']['failed']} failed, "
+        f"{result['summary']['skipped']} skipped"
+    )
+    return result
+
+
+def collect_main_files_for_templates(job_id: int, user: Any | None=None) -> None:
```

Ruff (0.14.10)

Unused function argument: `user`

(ARG001)